charts/jaeger/templates/cleanup-es-indexes-cronjob.yaml

```
@@ -0,0 +1,70 @@
+{{- if .Values.esIndexCleaner.enabled -}}
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: {{ include "jaeger.fullname" . }}-es-index-cleaner
+  labels:
+    {{- include "jaeger.labels" . | nindent 4 }}
+    app.kubernetes.io/component: es-index-cleaner
+{{- if .Values.esIndexCleaner.annotations }}
+  annotations:
+{{ toYaml .Values.esIndexCleaner.annotations | indent 4 }}
+{{- end }}
+spec:
+  concurrencyPolicy: "Forbid"
+  schedule: {{ .Values.esIndexCleaner.schedule | quote }}
+  successfulJobsHistoryLimit: {{ .Values.esIndexCleaner.successfulJobsHistoryLimit }}
+  failedJobsHistoryLimit: {{ .Values.esIndexCleaner.failedJobsHistoryLimit }}
+  suspend: false
+  jobTemplate:
+    spec:
+      template:
+        metadata:
+          labels:
+            {{- include "jaeger.selectorLabels" . | nindent 12 }}
+            app.kubernetes.io/component: es-index-cleaner
+            {{- if .Values.esIndexCleaner.podLabels }}
+{{ toYaml .Values.esIndexCleaner.podLabels | indent 12 }}
+            {{- end }}
+        spec:
+          serviceAccountName: {{ template "jaeger.esIndexCleaner.serviceAccountName" . }}
+          securityContext:
+{{ toYaml .Values.esIndexCleaner.podSecurityContext | indent 12 }}
+          containers:
+          - name: {{ include "jaeger.fullname" . }}-esIndexCleaner
```

```suggestion
          - name: {{ include "jaeger.fullname" . }}-es-index-cleaner
```

container name should be in kebab-cases

---

Man, i thought i changed it everywhere 😢

---

Made the change 🤞